File: microservices/feedback-service/src/main/java/com/karate/feedback_service/FeedbackServiceApplication.java
package com.karate.feedback_service;

import com.karate.feedback_service.infrastructure.jwt.JwtConfigurationProperties;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.springframework.cloud.openfeign.EnableFeignClients;

@SpringBootApplication
@EnableFeignClients
@EnableConfigurationProperties(value = {JwtConfigurationProperties.class})
public class FeedbackServiceApplication {

    public static void main(String[] args) {
        SpringApplication.run(FeedbackServiceApplication.class, args);
    }

}

File: microservices/feedback-service/src/main/kotlin/com/karate/feedback_service/FeedbackServiceApplication.kt
package com.karate.feedback_service;

import com.karate.feedback_service.infrastructure.jwt.JwtConfigurationProperties
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication
import org.springframework.cloud.openfeign.EnableFeignClients

@SpringBootApplication
@EnableFeignClients
@EnableConfigurationProperties(JwtConfigurationProperties::class)
class FeedbackServiceApplication
fun main(args: Array<String>) {
    runApplication<FeedbackServiceApplication>(*args)
}
